go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
 	github.com/invopop/jsonschema v0.6.0
 	github.com/jacobsa/go-serial v0.0.0-20180131005756-15cf729a72d4
-	github.com/jedib0t/go-pretty/v6 v6.4.6
+	github.com/jedib0t/go-pretty v4.3.0+incompatible
```

Comment:
Why are we downgrading this dependency to a weird `+incompatible` version? It looks like doing that also adds a number of other dependencies.

Reply:
I'm not sure how this changed.  I will revert.